### PR TITLE
merge log decryption feature branch into master

### PR DESCRIPTION
**Pull request by tibulca**
## Proposed changes

_Jira ticket:_ CLOUDP-#

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by blva — DISMISSED** [review]
LGTM

File: internal/cli/atlas/atlas.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func Builder() *cobra.Command {
 		events.Builder(),
 		metrics.Builder(),
 		performanceadvisor.Builder(),
-		logs.Builder(),
+		logs.MongoCLIBuilder(),
```
Comment (gssbzn) [inline internal/cli/atlas/atlas.go:89]: 👀  mongocli in atlas? also the commend on the method says "until atlas is released" but atlas was already released

**Review by gssbzn — COMMENTED** [review]
is this really ready to be merged to master? reading a lot of the commands are placeholders, which we are adding docs for and will be documented next time we release

If we want to merge this let's hide or disable the commands until they are releasable

File: internal/cli/atlas/atlas.go (hunk 1 above)
Comment (blva) [reply internal/cli/atlas/atlas.go:89]: I made the comment I think it should be "Until atlas commands are deprecated in MongoCLI". 

AFAIK this builder is used only by MongoCLI, hence the naming, fine w/ any naming though 👍

**Comment by tibulca** [comment]
> 

@gssbzn you are right, the atlas commands were included by mistake so I'm removing those

**Pull request closed without merging**